src/main/scala/ru/otus/sc/App.scala

```diff
@@ -1,22 +1,123 @@
 package ru.otus.sc
 
+import ru.otus.sc.countdown.dao.impl.CountdownDaoImpl
+import ru.otus.sc.countdown.model.{CountdownRequest, CountdownResponse}
+import ru.otus.sc.countdown.service.CountdownService
+import ru.otus.sc.countdown.service.impl.CountdownServiceImpl
+import ru.otus.sc.counter.dao.impl.CounterDaoImpl
+import ru.otus.sc.counter.model.{CounterRequest, CounterResponse}
+import ru.otus.sc.counter.service.CounterService
+import ru.otus.sc.counter.service.impl.CounterServiceImpl
+import ru.otus.sc.echo.dao.impl.EchoDaoImpl
+import ru.otus.sc.echo.model.{EchoRequest, EchoResponse}
+import ru.otus.sc.echo.service.EchoService
+import ru.otus.sc.echo.service.impl.EchoServiceImpl
 import ru.otus.sc.greet.dao.impl.GreetingDaoImpl
 import ru.otus.sc.greet.model.{GreetRequest, GreetResponse}
 import ru.otus.sc.greet.service.GreetingService
 import ru.otus.sc.greet.service.impl.GreetingServiceImpl
+import ru.otus.sc.reverse.dao.impl.ReverseDaoImpl
+import ru.otus.sc.reverse.model.{ReverseRequest, ReverseResponse}
+import ru.otus.sc.reverse.service.ReverseService
+import ru.otus.sc.reverse.service.impl.ReverseServiceImpl
+import ru.otus.sc.storage.dao.impl.StorageDaoImpl
+import ru.otus.sc.storage.model.{StorageRequest, StorageResponse}
+import ru.otus.sc.storage.service.StorageService
+import ru.otus.sc.storage.service.impl.StorageServiceImpl
+import ru.otus.sc.sum.dao.impl.SumDaoImpl
+import ru.otus.sc.sum.model.{SumRequest, SumResponse}
+import ru.otus.sc.sum.service.SumService
+import ru.otus.sc.sum.service.impl.SumServiceImpl
+
+// Helper class which aggregates service entries to single point
```

Comment:
```suggestion
/** Helper class which aggregates service entries to single point */
```

Подобный синтаксис подхватывается компилятором при генерации документации. Рекомендую использовать для методов, классов, членов классов и всего для чего нужен ScalaDoc.

Reply:
принято